modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/GridCacheDatabaseSharedManager.java

```diff
@@ -2863,11 +2869,16 @@ private RestoreLogicalState applyLogicalUpdates(
 
                         break;
 
-                    case MASTER_KEY_CHANGE_RECORD:
+                    case MASTER_KEY_CHANGE_RECORD_V2:
```

Comment:
Let's also keep `case MASTER_KEY_CHANGE_RECORD:`

Reply:
There is no compatibility issue. If we read "old" (MASTER_KEY_CHANGE_RECORD) in the serializer, we will have a new WALRecord with type V2 here.